strapi/strapi
# Feat/open Guided tour to all admins

### What does it do?

This PR opens the Guided Tour to all admins.
It also changes some text for the `super-admin` GT to be more straightforward.

### Why is it needed?

It is too bad that the current Guided Tour is only accessible for super-admins (developers). The idea is to also propose it to non-technical people discovering the product to guide them a little bit. 

### How to test it?

- Create a Super-Admin user
- Create a non-super admin user.

In development environment:
You should get 3 steps for both users which are different based on the user type.

In the production environment:
You should get 4 steps for both users. The extra step is the same step for both which is about inviting a user but it only make sense to have it on production.
